src/state/slices/actionSlice/selectors.ts

```diff
@@ -140,6 +164,13 @@ export const selectLimitOrderActionsByWallet = createDeepEqualOutputSelector(
   },
 )
 
+export const selectWalletGenericTransactionActionsSorted = createDeepEqualOutputSelector(
+  selectWalletActionsSorted,
+  actions => {
+    return actions.filter(isGenericTransactionAction)
```

Comment:
re: comment above, this works, as this shorthand with a single predicate call will keep the narrowing from the predicate